mamba/mamba/utils.py

```diff
@@ -211,6 +211,11 @@ def init_api_context(use_mamba_experimental=False):
     api_ctx.channels = context.channels
     api_ctx.platform = context.subdir
 
+    if "MAMBA_EXTRACT_THREADS" in os.environ:
+        max_threads = int(os.environ["MAMBA_EXTRACT_THREADS"])
+        api_ctx.extract_threads = max_threads
+        api.DownloadExtractSemaphore.set_max(max_threads)
```

Comment:
Ideally we don't have to call this function and use `Context` as the imperative configuration value. If we really have to we could call this function inside the Context constructor.

Reply:
I understand, having to call `set_verbosity` or other methods to apply values is then also not correct.
I'll have a look at what could be done without changing the API

Reply:
Verbosity is special because it also needs to call some functions for libcurl and libsolv to set their verbosities